modules/db-for-postgre-sql/flexible-server/README.md

```diff
@@ -806,3 +664,149 @@ module flexibleServer './db-for-postgre-sql/flexible-server/main.bicep' = {
 
 </details>
 <p>
+
+
+## Notes
+
+### Parameter Usage: `firewallRules`
+
+To enable firewall rules on the PostgreSQL flexible server:
+
+- Used when the desired connectivity mode is "Public Access" only.
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"firewallRules": {
+    // Example showing all available fields
+    "value": [
+        {
+            "name": "AllowAllWindowsAzureIps", //Use this rule to allow Trusted Azure services to access the server
+            "endIpAddress": "0.0.0.0",
+            "startIpAddress": "0.0.0.0"
+        },
+        {
+            "name": "test-rule1",
+            "startIpAddress": "10.10.10.1", //Start IP address for the firewall rule. Must be IPv4 format
+            "endIpAddress": "10.10.10.10" //End IP address for the firewall rule. Must be IPv4 format
+        }
+    ]
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+firewallRules:  [
+    // Example showing all available fields
+  {
+      name: 'AllowAllWindowsAzureIps', //Use this rule to allow Trusted Azure services to access the server
+      endIpAddress: '0.0.0.0'
+      startIpAddress: '0.0.0.0'
+  }
+  {
+      name: "test-rule1",
+      startIpAddress: '10.10.10.1' //Start IP address for the firewall rule. Must be IPv4 format
+      endIpAddress: '10.10.10.10' //End IP address for the firewall rule. Must be IPv4 format
+  }
+]
+```
+
+</details>
+<p>
+
+### Parameter Usage: `configurations`
```

Comment:
This is a child resource for which we have docs

Reply:
removed